Mirror/Runtime/NetworkAnimator.cs

```diff
@@ -238,25 +221,23 @@ void WriteParameters(NetworkWriter writer, bool autoSend)
                     continue;
 
                 AnimatorControllerParameter par = parameters[i];
-                if (par.type == AnimatorControllerParameterType.Int)
+                switch (par.type)
```

Comment:
please do this in a separate PR and use this one only for param array stuff or vice versa